src/app/(cities)/zielona-gora/page.mdx

```diff
@@ -0,0 +1,9 @@
+export const metadata = {
+  title: "meet.js Zielona Góra",
+};
+
+<div className="prose dark:prose-invert max-w-none space-y-8 my-8">
+  # Zielona Góra
```

Comment:
why change these in other cities? or is the branch in need of a rebase?

Reply:
rebase is needed